streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -443,6 +443,12 @@ public class StreamsConfig extends AbstractConfig {
     @SuppressWarnings("WeakerAccess")
     public static final String UPGRADE_FROM_40 = UpgradeFromValues.UPGRADE_FROM_40.toString();
 
+    /**
+     * Config value for parameter {@link #UPGRADE_FROM_CONFIG "upgrade.from"} for upgrading an application from version {@code 4.1.x}.
+     */
+    @SuppressWarnings("WeakerAccess")
+    public static final String UPGRADE_FROM_41 = UpgradeFromValues.UPGRADE_FROM_41.toString();
```

Comment:
Should we also add `UPGRADE_FROM_41` to `UPGRADE_FROM_DOC` accordingly?

Reply:
Ok I'll follow up with a PR

Reply:
Actually we should delete the `Accepted values are ...` from the description. The config is defined with a `Validator` so we already print the valid values in the docs: https://kafka.apache.org/documentation/#streamsconfigs_upgrade.from